models/iaf_psc_alpha_ps.cpp

```diff
@@ -268,15 +268,21 @@ nest::iaf_psc_alpha_ps::pre_run_hook()
 
   V_.h_ms_ = Time::get_resolution().get_ms();
 
-  V_.psc_norm_ex_ = 1.0 * numerics::e / P_.tau_syn_ex_;
-  V_.psc_norm_in_ = 1.0 * numerics::e / P_.tau_syn_in_;
+  // pre-compute inverse member variables
+  V_.inv_c_m_ = 1.0 / P_.c_m_;
+  V_.inv_tau_m_ = 1.0 / P_.tau_m_;
+  V_.inv_tau_syn_ex_ = 1.0 / P_.tau_syn_ex_;
+  V_.inv_tau_syn_in_ = 1.0 / P_.tau_syn_in_;
+
+  V_.psc_norm_ex_ = 1.0 * numerics::e * V_.inv_tau_syn_ex_;
```

Comment:
Why multiply by 1?

Reply:
I was going with the existing style. I didn't change the 1.0 thing that was already there.